test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -7372,11 +7480,63 @@ def forward(self, x):
         ort_sess1 = convert_to_onnx(model, input=(x,), opset_version=self.opset_version,
                                     training=torch.onnx.TrainingMode.TRAINING)
         ort_outs1 = run_ort(ort_sess1, input=(x,))
+        pt_outs1 = model(x)
+        [np.testing.assert_allclose(ort_out1, pt_out1.detach(), atol=1e-4, rtol=0.001) for ort_out1, pt_out1 in
```

Comment:
Change the format of rtol=0.001 to rtol=1e-3?

In addition, for atol, our normal testing uses 'atol=1e-7'. Should we keep the same one here?